package.json

```diff
@@ -23,13 +23,15 @@
     "storybook:build": "cross-env NODE_ENV=test build-storybook"
   },
   "dependencies": {
+    "@codemirror/language": "^6.3.1",
```

Comment:
all those new packages (do we really need them?) should be under `dependencies` and not scattered between `dependencies` and `devDependencies`